### PR TITLE
Stripe fix: hidden field were not working as product qty fields for purchases

### PR DESCRIPTION
**Pull request by ericpias**
Thought I/we tested this originally, but apparently not.  Needed for the ox.work plan selector demo.

### CODE REVIEW

File: src/integrations/stripe.ts [hunk 1]
```diff
@@ -33,25 +34,25 @@ export function installStripe(stripeConfig: any) {
 // Returns mapping of servar key (or hidden field key) to value
 export function getFlatStripeCustomerFieldValues(
   stripeConfig: any,
-  fieldValues: any
+  _fieldValues: any
```
Comment (bo-dun-1) [inline src/integrations/stripe.ts:37]: instead of passing fieldValues in everywhere, can you use it directly from the import?

Comment (ericpias) [reply src/integrations/stripe.ts:37]: Doing it this way makes it convenient for test cases.  I don't actually pass it at runtime and it does use the import at runtime.  It is only passed from the test case.

Comment (bo-dun-1) [reply src/integrations/stripe.ts:37]: i would prefer we don't structure production code around test case convenience. You can always initialize fieldValues at a global level (we do that for all other test cases)

Comment (ericpias) [reply src/integrations/stripe.ts]: Changes made.

**Review by bo-dun-1 — APPROVED** [review]

**Pull request merged**